doc/building.html

```diff
@@ -314,6 +314,7 @@ <h3 id="apple-xcode">Apple Xcode</h3>
 <li>Use configure option <code>--with-xcode-path</code>, e.g. <code>configure --with-xcode-path=/Applications/Xcode13.1.app</code> This allows using a specific Xcode version for an OpenJDK build, independently of the active Xcode version by <code>xcode-select</code>.</li>
 </ul>
 <p>If you have recently (inadvertently) updated your OS and/or Xcode version, and the JDK can no longer be built, please see the section on <a href="#problems-with-the-build-environment">Problems with the Build Environment</a>, and <a href="#getting-help">Getting Help</a> to find out if there are any recent, non-merged patches available for this update.</p>
+<p>Experimental support for XCode 14 can be enabled using the <code>--enable-xcode14</code> configuration flag.</p>
```

Comment:
Please remove this. We don't usually update `doc/building.html` for every build change.

Reply:
Removed in last commit